src/synthesizer/__init__.py

```
@@ -31,22 +29,14 @@
 # Make a version available at the top level
 from synthesizer._version import __version__
 
-# Make the openmp check available at the top level
+# Import things we want at the top level
+from synthesizer.emissions.line import LineCollection
```

Ruff (0.11.9)

`synthesizer.emissions.line.LineCollection` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)